web/components/liquidations-reward.tsx

```diff
@@ -29,6 +34,42 @@ export const LiquidationReward: React.FC<LiquidationRewardProps> = ({
   const [state, setState] = useContext(AppContext);
 
   const claim = async () => {
+
+    const postConditions = [];
+    if (tokenIsStx) {
+
+      // PC
+      postConditions.push(
+        makeContractSTXPostCondition(
+          contractAddress,
+          'arkadiko-liquidation-rewards-v1-1',
+          FungibleConditionCode.Equal,
+          uintCV(claimable).value,
+        )
+      )
+    } else {
+
+      // FT name
+      var tokenName = "diko";
+      Object.keys(tokenTraits).forEach((key, index) => {
+        if (tokenTraits[key].address == token.split('.')[0] && tokenTraits[key].swap == token.split('.')[1]) {
```

Comment:
NTH: use `filter` instead of a forEach and if statement

```
Object.keys(tokenTraits).filter(obj => obj.address === token...
```
something like that